### PR TITLE
Added remind confirm and apply routes

### PR DESCRIPTION
**Pull request by Barnes2197**

### CODE REVIEW

File: src/controllers/application.js [hunk 1]
```diff
@@ -480,6 +480,40 @@ const resetPassword = async (req,res) => {
       httpResponse.failureResponse(res,err);
     }
 }
+const remindConfirm = async (req, res) =>
+{
+  try{
+
+        const remind = await Applicant.find({applicationStatus : "accepted"})
+
+        for(let i = 0;i < remind.length;i++)
```
Comment (Tommy2016x) [inline src/controllers/application.js]: use array prototype (map, foreach, etc)instead of for loop

File: src/controllers/application.js [hunk 2]
```diff
@@ -480,6 +480,40 @@ const resetPassword = async (req,res) => {
...
+  httpResponse.failureResponse(res, e)
+}
+}
+
+const remindApply = async (req,res) =>
+{
+  try{
+    const remind = await Applicant.find({applicationStatus : "not applied"})
+
+        for(let i = 0;i < remind.length;i++)
```
Comment (Tommy2016x) [inline src/controllers/application.js]: same here

File: src/routes/index.js [hunk 3]
```diff
@@ -29,6 +29,8 @@ apiRouter.put("/application/apply",hackerAuthMiddleware, application.apply);
 apiRouter.put("/application/unconfirm", hackerAuthMiddleware, application.unconfirm);
 apiRouter.put("/application/forgot_password",application.forgotPassword);
 apiRouter.put("/application/reset_password",application.resetPassword);
+apiRouter.get("/application/remind_confirm",application.remindConfirm);
```
Comment (Tommy2016x) [inline src/routes/index.js]: put these under admin route and add the admin auth middleware

File: src/controllers/application.js [hunk 4]
```diff
@@ -470,6 +470,34 @@ const resetPassword = async (req,res) => {
       httpResponse.failureResponse(res,err);
     }
 }
+const remindConfirm = async (req, res) =>
+{
+  try{
+
+        const remind = await Applicant.find({applicationStatus : "accepted"})
+
+        remind.map(mailService.applied)
```
Comment (Tommy2016x) [inline src/controllers/application.js]: not proper implementation of map, 
ex:
remind.map(applicant => {
   //do stuff with each applicant object
})

File: src/controllers/application.js [hunk 5]
```diff
@@ -470,6 +470,34 @@ const resetPassword = async (req,res) => {
...
+  httpResponse.failureResponse(res, e)
+}
+}
+
+const remindApply = async (req,res) =>
+{
+  try{
+    const remind = await Applicant.find({applicationStatus : "not applied"})
+
+        remind.map(mailService.applied)
```
Comment (Tommy2016x) [inline src/controllers/application.js]: same here

**Comment by Barnes2197** [comment]
It should be good now

**Pull request merged**